Domains/4-Application/Connectors/ConceptStation/Released/CSPhysical.01.00.02.ecschema.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="CSPhysical" alias="csp" version="01.00.02" description="iModel Connector schema capturing application data from ConceptStation." xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2">
```

Comment:
During releases, we aim at keeping the WIP copy of the schema one minor version ahead of the one being released. Since the WIP copy is at 1.0.2 already and the previous release was 1.0.0, it seems the version of this release would rather be 1.0.1.

Reply:
I changed the release version to 01.00.01

Reply:
Please update it in the `SchemaInventory.json` as well.

Reply:
I updated the schema inventory.